build/windows/win_x86.nsi

```diff
@@ -144,62 +146,17 @@ Section "Runtime Environment" SecRuntime
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\Markdown-3.0.1-py2.py3-none-any.whl"'
-
-  # Install PyYAML.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\PyYAML-3.13-cp36-cp36m-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\PyYAML-3.13-cp36-cp36m-win_amd64.whl"'
-
-  # Install PyQt5.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\PyQt5_sip-4.19.13-cp36-none-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\PyQt5_sip-4.19.13-cp36-none-win_amd64.whl"'
-  File "${BUILD_DIR}\PyQt5-5.11.3-5.11.2-cp35.cp36.cp37.cp38-none-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\PyQt5-5.11.3-5.11.2-cp35.cp36.cp37.cp38-none-win_amd64.whl"'
-  File "${BUILD_DIR}\QScintilla-2.10.8-5.11.2-cp35.cp36.cp37.cp38-none-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\QScintilla-2.10.8-5.11.2-cp35.cp36.cp37.cp38-none-win_amd64.whl"'
-
-  # Install NumPy.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\numpy-1.13.1+mkl-cp36-cp36m-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\numpy-1.13.1+mkl-cp36-cp36m-win_amd64.whl"'
-
-  # Install SciPy.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\scipy-0.18.1-cp36-cp36m-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\scipy-0.18.1-cp36-cp36m-win_amd64.whl"'
-
-  # Install pyshp.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\pyshp-1.2.10.tar.gz"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\pyshp-1.2.10.tar.gz"'
-
-  # Install ruamel.yaml.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\ruamel.yaml-0.15.58.tar.gz"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\ruamel.yaml-0.15.58.tar.gz"'
-
-  # Install psutil.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\psutil-5.4.8-cp36-cp36m-win_amd64.whl"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\psutil-5.4.8-cp36-cp36m-win_amd64.whl"'
-
-  # Install pyDes.
-  SetOutPath $INSTDIR\prerequisites
-  File "${BUILD_DIR}\pyDes-2.0.1.tar.gz"
-  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install "$INSTDIR\prerequisites\pyDes-2.0.1.tar.gz"'
+  # Install python packages.
+  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install --upgrade pip'
+  ExecWait '"$PYTHON_SCRIPTS\python.exe" -m pip install Markdown PyYAML PyQt5 PyQtWebEngine QScintilla numpy scipy pyshp ruamel.yaml psutil pyDes bgem'
 
   # Install gmsh.
   SetOutPath $INSTDIR
   File /r "${BUILD_DIR}\gmsh"
 
   # Install intersections.
-  SetOutPath $INSTDIR
-  File /r "${GIT_DIR}\submodules\intersections"
+  #SetOutPath $INSTDIR
+  #File /r "${GIT_DIR}\submodules\intersections"
 
   # Install yaml_converter.
   SetOutPath $INSTDIR
```

Comment:
Set flow123d installer to version 3.9.0.
Cleanup NSIS file. E.g: SecJobScheduler